apps/api/src/intern/dto/createIntern.dto.ts

```diff
@@ -0,0 +1,26 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { Field, Prisma } from '@prisma/client';
+import { IsEmail, IsString, MinLength } from 'class-validator';
+
+export class CreateInternDto {
```

Comment:
Jesi pokušao ovo u packages/types staviti? Jedna od većih prednosti ovakvog stacka je da možemo dijeliti tipove na fe i be

Reply:
proba, nije uspilo :(